benchmarks/deconvolution-1D/server.py

```diff
@@ -0,0 +1,139 @@
+import umbridge
+import numpy as np
+import cuqi
+
+
+class Deconvolution1D(umbridge.Model):
```

Comment:
For clarity I would rename this to avoid a name clash with the CUQIpy testproblem itself.